packages/expo-image/src/Image.tsx

```diff
@@ -198,14 +198,14 @@ export class Image extends React.PureComponent<ImageProps> {
       );
       loggedDefaultSourceDeprecationWarning = true;
     }
-
+    // @ts-expect-error
     if (restProps.children && !loggedRenderingChildrenWarning) {
       console.warn(
         'The <Image> component does not support children. If you want to render content on top of the image, consider using the <ImageBackground> component or absolute positioning.'
       );
       loggedRenderingChildrenWarning = true;
     }
-
+    
```

Comment:
Linter is failing on this 😉 Don't forget to rebuild files after this is applied (not sure if this is needed for just whitespace changes, probably not).
```suggestion

```